utils/_context/containers.py

```diff
@@ -516,9 +516,10 @@ def __init__(self, host_log_folder) -> None:
 
 class OpenTelemetryCollectorContainer(TestedContainer):
     def __init__(self, host_log_folder) -> None:
+        image = os.environ.get("OTEL_COLLECTOR_IMAGE", "otel/opentelemetry-collector-contrib:latest")
```

Comment:
Could you just change then env var name to `SYSTEM_TESTS_OTEL_COLLECTOR_IMAGE` ? I'm expecting soon or later some trouble with env var name conlfict ...

```suggestion
        image = os.environ.get("SYSTEM_TESTS_OTEL_COLLECTOR_IMAGE", "otel/opentelemetry-collector-contrib:latest")
```

Reply:
SGTM